frontend/src/components/EntertainmentButton/styles.css

```
@@ -0,0 +1,9 @@
+.text-button-style {
+  font-family: "Roboto";
+  font-size: 24px;
+  color: #A61700;
+}
+
+.button-style {
+  background-color: #FFBA73;
+}
```

Let's get rid of `-style` postfix everywhere

```suggestion
.text-button {
  font-family: "Roboto";
  font-size: 24px;
  color: #A61700;
}

.button {
  background-color: #FFBA73;
}
```